pkg/apiserver/common/common.go

```diff
@@ -67,25 +66,26 @@ func NewService(
 func Register(r *gin.RouterGroup, s *Service) {
 	endpoint := r.Group("/common")
 
-	endpoint.GET("/pods/all", s.GetPods)
-	endpoint.GET("/namespaces/all", s.GetNamespaces)
-	endpoint.GET("/kinds/all", s.GetKinds)
+	endpoint.GET("/pods", s.getPods)
+	endpoint.GET("/namespaces", s.getNamespaces)
+	endpoint.GET("/kinds", s.getKinds)
 
 }
 
-// GetPods returns the list of pods
-func (s *Service) GetPods(c *gin.Context) {
-	pods :=  make([]Pod, 0)
+// @Summary Get all pods from Kubernetes cluster.
+// @Description Get all pods from Kubernetes cluster.
+// @Produce json
+// @Success 200 {array} Pod
+// @Router /common/pods [get]
+// @Failure 401 {object} utils.APIError "Unauthorized failure"
+func (s *Service) getPods(c *gin.Context) {
+	pods := make([]Pod, 0)
 
 	var podList v1.PodList
 	err := s.kubeCli.List(context.Background(), &podList)
```

Comment:
Just come across another thing. Maybe we need to support an optional `namespace` parameter in this API.